lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -44,9 +43,6 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
   })();
 
   const message = (() => {
-    if (invalidScope) {
-      return null;
-    }
```

Comment:
There's a couple of thing going on here:

- Now for the `invalidScope` equivalent it shows  `Something went wrong when authorizing Hypothesis` but the alternative, the null value,  displays `An unknown error occurred.` which is arguably worse. 

    I think this should be solved while tackling: https://github.com/hypothesis/lms/issues/2997 and change it to "null" not displaying any message.

- This is now a function that contains only a return statement. I think this way is easier to change/extend later but I'm happy changing it back to just a constant value.

Reply:
I would suggest to change this to a constant value, as I think it is pretty obvious how it could be changed to handle other values in future. This isn't one of those scenarios where a lot of effort has gone into the scaffolding that needs to be preserved for the future.

Reply:
You are right, I was thinking the next PR based on this one would change it but it actually uses a different file. Changed it to a const value.